com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -887,6 +902,17 @@ void GenerateShaderPass(int targetIndex, PassDescriptor pass, ActiveFields activ
                 spliceCommands.Add("Debug", debugBuilder.ToCodeBlock());
             }
 
+            // --------------------------------------------------
+            // Additional Commands
+
+            if (pass.additionalCommands != null)
```

Comment:
Introduce the AdditionalCommand descriptors to unlock VFX to splice in all of the complex work it does for property mapping between functions evaluation and assignments.